src/stats/hist.jl

```diff
@@ -28,6 +28,13 @@ can be normalized by setting `normalization`. Possible values are:
    norm 1.
 *  `:none`: Do not normalize.
 
+The following attributes can move the histogram around,
+which comes in handy when placing multiple histograms into one plot:
+* offset = 0.0: adds an offset to every value
+* fillto = 0.0: defines where the bar starts
```

Comment:
I'm not too sure I like this API, in that for things to work you always need to set both to the same value. Wouldn't it be easier to just add `offset` to `barplot`, which would make the bar start at `offset` but (unlike `fillto`) have length `ys`? That way, users only need to set `offset` in histograms.

If we are unsure that `offset` makes sense for `barplot`, we can add it only for `hist`, but I think "shift bar while preserving the length" is a clearer meaning.